training/cleanup_artifacts.py

```diff
@@ -24,7 +24,7 @@
 api = wandb.Api()
 
 DEFAULT_PROJECT = "fsdl-text-recognizer-2021-training"
-DEFAULT_ENTITY = api.default_entity
+DEFAULT_ENTITY = "fullstackdeeplearning"   # api.default_entity
```

Comment:
my `api.default_entity` was `tiiai`, so felt it is better to be explicit

Reply:
you're right that we don't want to be surprising users when deleting data.

but the FSDL entity won't be accessible to a generic wandb user, so this choice means the script will fail with default arguments.

instead, i think we should ask folks to explicitly opt in to using the DEFAULT entity, so they at least know what they are doing.

it's plausible to me that your default entity was particularly surprising because of an old W&B bug that caused users' entities to automatically update when they joined a new entity, which may have been in place when you joined `tiiai`.